sw/device/lib/base/memory.h

```diff
@@ -19,6 +19,22 @@
 
 #include "sw/device/lib/base/macros.h"
 
+// When compiling unit tests on the host machine, we must mangle the names of
+// OpenTitan's memory functions to avoid conflicting with the libc variants. For
+// instance, without name mangling, the compiler may complain that `memcpy()`
+// has a different exception specifier than the variant declared in <string.h>.
+// (The <string.h> include is unavoidable because it's pulled in by GoogleTest.)
+//
+// Separately, ASan segfaults during initialization when it calls our
+// user-defined memory functions. Prefixing the troublesome functions works
+// around this ASan bug, enabling us to run the unit tests with ASan enabled.
+// See https://github.com/lowRISC/opentitan/issues/13826.
+#ifdef OT_PREFIX_MEM_FUNCTIONS
+#define MANGLE(name) ot_##name
+#else
+#define MANGLE(name) name
+#endif
+
```

Comment:
WDYT about slightly modifying this to become something like `OT_PREFIX_IF_NOT_RV32(name)` that could check `OT_PLATFORM_RV32`?

Reply:
It's a little verbose, but it definitely clarifies the intent at each call site. Done.